gateway/envs.go

```diff
@@ -32,12 +33,18 @@ func (g *Gateway) GetEnvs(ctx context.Context, req *entity.GetEnvsRequest) (*ent
 	return resp.Envs, nil
 }
 
-func (g *Gateway) UpsertVariablesFromObject(ctx context.Context, req *entity.UpdateEnvsRequest) error {
-	gqlReq, err := g.NewRequestWithAuth(`
+func (g *Gateway) UpdateVariablesFromObject(ctx context.Context, req *entity.UpdateEnvsRequest, replace bool) error {
+	queryName := "upsertVariablesFromObject"
+
+	if replace {
+		queryName = "variablesSetFromObject"
+	}
```

Comment:
Kinda a potential foot gun. I know I gave you the "replace" verbiage but it's one of those things where I'd hate someone to call this function unknowingly, have a release get cut, and then accidently delete all their variables :/

```suggestion
	if replace {
	        // When replacing, use the set query which will blow away all old variables and only set the ones in this query
		queryName = "variablesSetFromObject"
	}
```

Reply:
I don't really understand what you are implying with your suggested change. Just a comment for somebody working on the CLI code or something beyond that?